cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js

```diff
@@ -0,0 +1,64 @@
+import "@testing-library/cypress/add-commands"
+import {generateEpoch} from '../../../support/tradersHub'
+
+function createSiblingacct(i){
+    cy.findByTestId('dt_currency-switcher__arrow').click()
+    cy.findByRole('button', { name: 'Add or manage account' }).click()
+    cy.get('.currency-list__item').eq(i).click()
+    cy.findByRole('button', { name: 'Add account' }).click()
+    cy.findByRole('heading', { name: 'Success!' }).should('be.visible')
+    cy.findByRole('button', { name: 'Deposit now' }).should('be.enabled')
+    cy.findByRole('button', { name: 'Maybe later' }).click()  
+}
+function getCurrencyList() {
+  return cy.get('div.currency-list__items').find('label').then(($labels) => { 
+  const labelCount = $labels.length
+  return labelCount
+  })
+}
+describe("QATEST-5797, QATEST-5820 - Add siblings accounts", () => {
+    const epoch = generateEpoch()
+    const sign_up_mail = `sanity${epoch}@deriv.com`
+    let country = Cypress.env("countries").CO
+    let nationalIDNum = Cypress.env("nationalIDNum").CO
+    let taxIDNum = Cypress.env("taxIDNum").CO
+    let currency = Cypress.env("accountCurrency").USD
+    beforeEach(() => {
+      localStorage.setItem("config.server_url", Cypress.env("stdConfigServer"))
+      localStorage.setItem("config.app_id", Cypress.env("stdConfigAppId"))
+      cy.c_visitResponsive("/endpoint", "desktop")
+      cy.findByRole("button", { name: "Sign up" }).should("not.be.disabled")
+      cy.c_enterValidEmail(sign_up_mail)
```

Comment:
we have a new command for this now  as `cy.c_setEndpoint(signUpMail)`
```suggestion
      cy.c_setEndpoint(signUpMail)
```